Ginger/Ginger/Agents/AgentEditPage.xaml

```diff
@@ -57,7 +57,7 @@
                     </StackPanel>
                     <StackPanel Orientation="Horizontal" VerticalAlignment="Center" Margin="0,10,0,0" x:Name="xVirtualAgentsPanel" Visibility="Visible">
                         <StackPanel  Orientation="Horizontal" VerticalAlignment="Center">
-                            <Label Style="{StaticResource $LabelStyle}" Content="Virutalisation :" Margin="0,0,0,0" VerticalAlignment="Center"/>
+                            <Label Style="{StaticResource $LabelStyle}" Content="Visualization :" Margin="0,0,0,0" VerticalAlignment="Center"/>
```

Comment:
This is for Virtual agent. So Visualization is not correct :)

Please update the spelling to "Virtualization"